service/controller/internal/cloudconfig/tcnp_test.go

```diff
@@ -29,6 +30,9 @@ import (
 //
 //  go test ./service/controller/internal/cloudconfig -run Test_Controller_CloudConfig_TCNP_Template_Render -update
 //
+
+var update = flag.Bool("update", false, "update .golden CF template file")
+
```

Comment:
Let's move this up please so that the godoc for the test function is in place again.